components/python/py3c/Makefile

```diff
@@ -0,0 +1,66 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 Klaus Ziegler
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=           py3c
+COMPONENT_VERSION=        1.3
+COMPONENT_PROJECT_URL=    https://pypi.org/project/py3c/#files
+COMPONENT_FMRI=           library/python/py3c
+COMPONENT_CLASSIFICATION= Development/Python
+COMPONENT_LICENSE=        MIT
+
+GIT=git
+GIT_REPO=https://github.com/encukou/py3c.git
```

Comment:
Why do you want master branch of the repo instead of using standard pypi tarball? You could set `COMPONENT_ARCHIVE_URL=	$(call pypi_url)`
Why don't you use BUILD_STYLE=setup.py ?

Reply:
Anyway, you could just download 1.3.1 version from github as tarball, this way build would deliver the same version whenever it's rebuilt.